### PR TITLE
Squash remaining unmerged changes and resubmit PR from #2758

### PR DESCRIPTION
**Pull request by petemounce**
@dslomov here you go. From #2758.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by mhlopko** [comment]
Attribution issue is gone, feel free to merge, if lgtmed.

**Comment by petemounce** [comment]
@dslomov please merge?

File: scripts/packages/chocolatey/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -15,19 +15,20 @@ $packageDir = Split-Path -parent $toolsDir
...
   -Url64bit "$($paramsText[0])" `
   -Checksum64 "$($paramsText[1])" `
-  -Checksum64Type "sha256" `
+  -ChecksumType64 "sha256" `
   -UnzipLocation "$packageDir"
 
 write-host "Ensure that msys2 dll is present in PATH to allow bazel to be run from non-msys2 shells"
 $pp = Get-PackageParameters $env:chocolateyPackageParameters
-Install-ChocolateyPath -PathToInstall "$($pp.msys2Path)\usr\bin" -PathType "Machine"
+$msys2Path = ($pp.msys2Path, "c:\tools\msys64" -ne $null)[0]
```
Comment (dslomov) [inline scripts/packages/chocolatey/tools/chocolateyinstall.ps1]: How does this syntax work?

Comment (petemounce) [inline scripts/packages/chocolatey/tools/chocolateyinstall.ps1]: That's idiomatic PowerShell null-coalescing, that is. Expressive, huh? :cry:

"Take the first item from the array that is not equal to null".

Comment (dslomov) [inline scripts/packages/chocolatey/tools/chocolateyinstall.ps1]: ok.

**Pull request closed without merging**